src/components/Header.css

```diff
@@ -148,7 +148,7 @@
         color: rgb(243, 55, 55);
     }
     100% {
-        color: rgb(207, 206, 145);
```

Comment:
either color will work haha it was just a test but i still have way to edit :P another conflict lol